src/Component/Sender/Adapter/AdapterInterface.php

```diff
@@ -26,6 +26,8 @@ public function send(
         EmailInterface $email,
         array $data,
         array $attachments = [],
-        array $replyTo = []
+        array $replyTo = [],
+        array $ccRecipients = [],
```

Comment:
Let's add it to the new interface, which will extend this one. Then keep existing logic and propagate with additional data if `cc` of `bcc` recipients are part of the contract.